gax-java/gax/clirr-ignored-differences.xml

```diff
@@ -106,16 +106,26 @@
     <className>com/google/api/gax/batching/Batcher</className>
     <method>*</method>
   </difference>
-  <!-- Ignore this as this was part of s2a-grpc ExperimentalApi revert -->
+  <!-- Ignore abstract method addition to an EndpointContext (InternalApi) -->
   <difference>
-    <differenceType>7002</differenceType>
-    <className>com/google/api/gax/rpc/FixedTransportChannelProvider</className>
-    <method>* withUseS2A(*)</method>
+    <differenceType>7013</differenceType>
+    <className>com/google/api/gax/rpc/EndpointContext</className>
+    <method>* useS2A()</method>
   </difference>
-  <!-- Ignore this as this was part of s2a-grpc ExperimentalApi revert -->
+  <!-- Ignore method addition to TransportChannelProvider interface (InternalExtensionOnly) -->
   <difference>
-    <differenceType>7002</differenceType>
+    <differenceType>7012</differenceType>
     <className>com/google/api/gax/rpc/TransportChannelProvider</className>
     <method>* withUseS2A(*)</method>
   </difference>
+   <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/api/gax/rpc/TransportChannelProvider</className>
+    <method>* needsMtlsEndpoint()</method>
+  </difference>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/api/gax/rpc/TransportChannelProvider</className>
+    <method>* withMtlsEndpoint(*)</method>
+  </difference>
```

Comment:
I think these can potentially be added in a future date (if needed).

Reply:
Ah yes, thanks for catching. Done in https://github.com/googleapis/sdk-platform-java/pull/3548/commits/a814a5371f845b20ee0d9a23bbd01266833dccb6